docs/deployment/settings.md

```diff
@@ -311,6 +311,7 @@ You can configure the Kyuubi properties in `$KYUUBI_HOME/conf/kyuubi-defaults.co
 | kyuubi.frontend.thrift.backoff.slot.length             | PT0.1S            | Time to back off during login to the thrift frontend service.                                                                                                                                                                                                                                                                                                                                                                                                                                                               | duration | 1.4.0 |
 | kyuubi.frontend.thrift.binary.bind.host                | &lt;undefined&gt; | Hostname or IP of the machine on which to run the thrift frontend service via the binary protocol.                                                                                                                                                                                                                                                                                                                                                                                                                          | string   | 1.4.0 |
 | kyuubi.frontend.thrift.binary.bind.port                | 10009             | Port of the machine on which to run the thrift frontend service via the binary protocol.                                                                                                                                                                                                                                                                                                                                                                                                                                    | int      | 1.4.0 |
+| kyuubi.frontend.thrift.binary.ssl.bind.port            | 10011             | Port of the machine on which to run the thrift frontend service via the binary protocol with SSL enabled.                                                                                                                                                                                                                                                                                                                                                                                                                   | int      | 1.7.0 |
```

Comment:
kyuubi.frontend.thrift.binary.bind.port also can be used for ssl, how do we suppose users to understand?